src/NetCoreApp1.Tests/Log4NetProviderExtensionsShould.cs

```diff
@@ -1,4 +1,9 @@
-﻿namespace NetCore1.Tests
+﻿using System.Reflection;
```

Comment:
To be consistent, can you move the using statements into the namespace?

Reply:
Sure